src/bin/index.ts

```diff
@@ -39,11 +40,22 @@ export async function bin(args: string[]) {
 	const { values } = parseArgs({
 		args,
 		options: {
+			help: {
+				short: "h",
+				type: "boolean",
+			},
 			mode: { type: "string" },
 		},
 		strict: false,
 	});
 
+	const help = values.help;
+
+	if (help) {
+		logHelpText("");
```

Comment:
> What is the best approach to fetch the version number?

I'd probably try with a combination of:

* The APIs and techniques mentioned in https://blog.logrocket.com/alternatives-dirname-node-js-es-modules to get the path to the package's own `package.json` file, relative to whatever file in `lib/`/`src/` is asking for it
* The existing `readFileSafeAsJson` helper to read in JSON files

Reply:
Thank you, I'll try that!

Reply:
I've implemented this based on the URL method!